core/api.go

```diff
@@ -99,6 +99,7 @@ func (h *Handler) init() {
 	h.actionDecoders = map[string]func(data []byte) (txbuilder.Action, error){
 		"control_account":                h.Accounts.DecodeControlAction,
 		"control_program":                txbuilder.DecodeControlProgramAction,
+		"control_with_receiver":          txbuilder.DecodeControlWithReceiverAction,
```

Comment:
After futzing with it, I decided that it's too much of a microchange to humanize the action labels here. "control_with_account" is preferable to "control_account", but the value prop is too limited to justify updating the wire protocol and SDKs. So `control_receiver` it is.